### PR TITLE
Mark zoom alias as turing instructor

### PR DESCRIPTION
**Pull request by AlecKap**
____ Wrote Tests __x__ Implemented __x__ Reviewed

## Necessary checkmarks:

- [x] All Tests are Passing in all environments

- [x] The code will run locally

## Type of change

- [ ] New feature
- [x] Bug Fix
- [x] Refactor

## Description of Change:

    description: Refactor #student_attendances so that student with name "Instructor" will not be rendered in student list on attendance show page.

## Requested feedback:

    I'd like feedback on...

## Check the correct boxes

- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes

- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

- [x] My code has no unused/commented out code
- [x] My code has no binding.pry's
- [x] I have reviewed my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -1 +1,2 @@
 --require spec_helper
+--format documentation
```
Comment (AlejandroLopez1992) [inline .rspec]: Let's have this option available locally but not passed up to the respository.

File: app/facades/attendance_show_facade.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@ def unassigned_zoom_aliases
   end
 
   def student_attendances
-    @attendance.student_attendances.includes(:student).by_attendance_status
+    @attendance.student_attendances.includes(:student).where.not(students: { name: "Instructor" }).by_attendance_status
```
Comment (AlejandroLopez1992) [inline app/facades/attendance_show_facade.rb:24]: Good solution, Let's create some testing for it to ensure it works exactly as intended.

File: db/seeds.rb [hunk 3]
```diff
@@ -1,9 +1,324 @@
+puts "\n== Seeding the database with fake data created by Turing's team RubyHive. Thank you for your patience. ==\n\n"
```
Comment (AlejandroLopez1992) [inline db/seeds.rb:1]: Thank you for creating seed data that will allow for easier localhost user testing in the future.

File: app/views/user/attendances/show.html.erb [hunk 4]
```diff
@@ -4,7 +4,9 @@
 <h3><%= button_to "Retake Attendance", attendance_retake_path(@attendance), class: 's-button' %></h3>
```
Comment (AlejandroLopez1992) [inline app/views/user/attendances/show.html.erb:4]: I like the further separation of each partial pages functionality, and then displaying each of those pages indicidually within the attendances/show page. It looks cleaner this way. Great job!

**Review by AlejandroLopez1992 — CHANGES_REQUESTED** [review]

File: app/facades/attendance_show_facade.rb (hunk 2 above)
Comment (AlecKap) [reply app/facades/attendance_show_facade.rb:24]: Thank you, and yes great call. We can add some facade tests as well as feature tests.

File: .rspec (hunk 1 above)
Comment (AlecKap) [reply .rspec]: Good idea, I will look to set this up locally.

**Review by AlejandroLopez1992 — APPROVED** [review]

**Pull request merged**